webpack/scenes/ContentViews/Details/ContentViewDetails.js

```diff
@@ -81,31 +81,17 @@ export default () => {
   if (status === STATUS.PENDING) return (<Loading />);
   if (status === STATUS.ERROR) return (<EmptyStateMessage error={error} />);
 
-  const dropDownItems = [
-    <DropdownItem
-      key="copy"
-      ouiaId="cv-copy"
-      onClick={() => {
-        setCopying(true);
-      }}
-    >
-      {__('Copy')}
-    </DropdownItem>,
-    <DropdownItem
-      key="delete"
-      ouiaId="cv-delete"
-      onClick={() => {
-        setDeleting(true);
-      }}
-    >
-      {__('Delete')}
-    </DropdownItem>,
-  ];
-
   const {
-    name, composite, permissions, environments, versions,
+    name, composite, rolling, permissions, environments, versions,
     generated_for: generatedFor, import_only: importOnly,
   } = details;
+
+  const dropDownItems = [];
+  if (!rolling) {
+    dropDownItems.push(<DropdownItem key="copy" ouiaId="cv-copy" onClick={() => { setCopying(true); }} > {__('Copy')}</DropdownItem>);
```

Comment:
Why is copying not supported for rolling CVs?

![image](https://github.com/user-attachments/assets/c410cc09-0e1b-4917-b961-7d5b4e35101b)

Reply:
To be honest, I guess we considered it as a follow-up feature (if needed).

Reply:
Fine by me.